src/rust/engine/process_execution/src/docker.rs

```diff
@@ -92,6 +92,16 @@ struct ImagePullCache {
   image_pull_policy: ImagePullPolicy,
 }
 
+fn docker_platform_identifier(platform: &Platform) -> String {
```

Comment:
Should probably return `&str`: can imagine cases where this is only used for comparison, for example.

Reply:
Done.